package.json

```diff
@@ -4,16 +4,15 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "coverage": "npm run db-migrate && mocha --reporter mochawesome --timeout=20000 --require babel-register --exit",
+    "coverage": "nyc report --reporter=text-lcov | coveralls",
     "db-migrate": "sequelize db:migrate",
     "db:undo-migration": "sequelize db:migrate:undo:all",
     "lint": "eslint routes models config test index.js",
-    "test": "npm run db:undo-migration && npm run db-migrate && mocha test/index.js --timeout=20000 --require babel-register --exit",
-    "start": "sequelize db:migrate && node dist/index",
+    "start": "sequelize db:migrate && npm run build && node index.js",
```

Comment:
@kleva-j can you explain where the app is being served from based on this start script?

Reply:
I was a bit distracted when I was fixing the code. I was supposed to serve/start the app from the index file in the dist folder.